apps/ui/src/global/types.ts

```diff
@@ -29,16 +29,6 @@ export interface Application {
 	updatedAt: string;
 	expiresAt: string;
 }
-export interface ApplicationWithApplicantInformation extends Application {
```

Comment:
These changes (and related changes in `data-model` / `manage/applications`) are also in :https://github.com/Pan-Canadian-Genome-Library/daco/pull/153
Intent is to make types that are shared and reusable btwn ui/api